tests/integration/components/paper-input-test.js

```diff
@@ -46,8 +45,9 @@ module('Integration | Component | paper input', function(hooks) {
 
     await render(hbs`{{paper-input iconComponent="custom-icon" icon="person" onChange=dummyOnChange}}`);
 
-    assert.equal(this.$('md-input-container md-icon').length, 0, 'default icon component is not rendered');
-    assert.equal(this.$('md-input-container .custom-icon').length, 1, 'custom icon component rendered');
+    assert.dom('md-input-container md-icon').doesNotExist()
+    assert.dom('md-input-container .custom-icon').exists()
```

Comment:
This should be:

```
assert.dom('md-input-container .custom-icon').exists({ count: 1 }, 'custom icon component rendered');
```

- try to keep assertion messages
- use `exists: 1` to make sure only one item is rendered
- keep semicolon `;`